.automation/build_archi_diagram.py

```diff
@@ -0,0 +1,77 @@
+# !/usr/bin/env python3
```

Comment:
Don't forget to mark scripts executable (e.g., `chmod u+x .automation/build_archi_diagram.py`) since otherwise the shebang line is fairly useless.